src/sentry/static/sentry/app/views/alerts/details/body.tsx

```diff
@@ -36,6 +36,19 @@ type Props = {
 } & RouteComponentProps<{alertId: string; orgId: string}, {}>;
 
 export default class DetailsBody extends React.Component<Props> {
+  /**
+   *  Return a string representing the time in seconds between two incident data points
+   */
+  calculateTimeWindowString(incident: Incident) {
+    if (incident.eventStats.data.length >= 2) {
+      return `${incident.eventStats.data[1][0] - incident.eventStats.data[0][0]}s`;
+    } else if (incident.alertRule) {
+      return `${incident.alertRule.timeWindow}m`;
+    } else {
+      return '10m';
+    }
```

Comment:
The `else` here isn't necessary because we are `return`ing (we have a lint rule coming that enforces this).

```suggestion
    if (incident.eventStats.data.length >= 2) {
      return `${incident.eventStats.data[1][0] - incident.eventStats.data[0][0]}s`;
    }
    
    if (incident.alertRule) {
      return `${incident.alertRule.timeWindow}m`;
    }
    
    return '10m';
```